src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+/*text*/
+$cards-row: 4;
+$header-accent: #00acdc;
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
```

Comment:
The $blue-accent variable has the same value as the $header-accent variable. Consider reusing the $header-accent variable instead, or remove the duplicate if it's not necessary.